### PR TITLE
support speccify image names of the sidecar & agent-initializer

### PR DESCRIPTION
**Pull request by zhao-kun**

### CODE REVIEW

File: operator/main.go [hunk 1]
```diff
@@ -80,6 +86,10 @@ type ConfigSpec struct {
 	CertDir              string   `yaml:"cert-dir" jsonschema:"required"`
 	CertName             string   `yaml:"cert-name" jsonschema:"required"`
 	KeyName              string   `yaml:"key-name" jsonschema:"required"`
+	Log4jConfigName      string   `yaml:"log4j-config-name" jsonschema:"required"`
+
+	EaseagentInitializerImageName string `yaml:"easeagent-initializer-image-name" jsonschema:"required"`
```
Comment (xxx7xxxx) [inline operator/main.go:91]: Can we use a general name such as `AgentInitializerImageName `, from a product view, we should not bind with EaseAgent.

File: operator/pkg/base/base.go [hunk 2]
```diff
@@ -18,6 +18,10 @@ type (
 		ImageRegistryURL string
 		ImagePullPolicy  string
 		SidecarImageName string
+		// The image name of the easeagent initializer
```
Comment (xxx7xxxx) [inline operator/pkg/base/base.go:21]:
```suggestion
		// AgentInitializerImageName is the image name of the Agent initializer.
```

File: operator/pkg/base/base.go [hunk 3]
```diff
@@ -18,6 +18,10 @@ type (
...
+		EaseagentInitializerImageName string
+		// Log4jConfigName default is easeagent-log4j.xml
```
Comment (xxx7xxxx) [inline operator/pkg/base/base.go:23]:
```suggestion
		// Log4jConfigName is  the name of log4f config name.
```

The default value is not suitable to comment here.

**Comment by zhao-kun** [comment]
Updated! `gofmt` check passed

**Review by xxx7xxxx — APPROVED** [review]

**Pull request merged**